apps/studio/components/interfaces/DiskManagement/ui/DiskManagement.constants.tsx

```diff
@@ -0,0 +1,168 @@
+  },
+  [DiskType.IO2]: {
+    storage: 0.195, // per GB per month
+    iops: 0.119, // per IOPS per month
+  },
+}
+
+export const DISK_LIMITS = {
+  [DiskType.GP3]: {
+    minStorage: 1,
+    maxStorage: 16384,
+    minIops: 3000,
+    maxIops: 16000,
+    minThroughput: 125,
+    maxThroughput: 1000,
+    includedIops: 3000,
+    includedThroughput: 125,
+  },
+  [DiskType.IO2]: {
+    minStorage: 4,
+    maxStorage: 61440,
+    minIops: 100,
+    maxIops: 256000,
+    includedIops: 0,
+    includedThroughput: 0,
+  },
+}
+
+export const DISK_TYPE_LABELS = {
+  [DiskType.GP3]: 'General Purpose SSD (gp3)',
+  [DiskType.IO2]: 'Provisioned IOPS SSD (io2)',
+}
+
+interface PlanDetails {
+  includedDiskGB: { gp3: number; io2: number }
+}
+
+export const PLAN_DETAILS: Record<PlanType, PlanDetails> = {
+  free: { includedDiskGB: { gp3: 1, io2: 0 } },
+  pro: { includedDiskGB: { gp3: 8, io2: 0 } },
+  team: { includedDiskGB: { gp3: 8, io2: 0 } },
+  enterprise: { includedDiskGB: { gp3: 8, io2: 0 } },
+}
+
+// export const COMPUTE_SIZE_MAX_IOPS = {
+//   ci_micro: 11800,
+//   ci_small: 11800,
+//   ci_medium: 11800,
+//   ci_large: 4750,
+//   ci_xlarge: 20000,
+//   ci_2xlarge: 20000,
+//   ci_4xlarge: 20000,
+//   ci_8xlarge: 40000,
+//   ci_12xlarge: 50000,
+//   ci_16xlarge: 80000,
+// }
+
+// export const COMPUTE_SIZE_MAX_THROUGHPUT = {
+//   ci_micro: 260.625,
+//   ci_small: 260.625,
+//   ci_medium: 260.625,
+//   ci_large: 593.75,
+//   ci_xlarge: 593.75,
+//   ci_2xlarge: 593.75,
+//   ci_4xlarge: 593.75,
+//   ci_8xlarge: 1187.5,
+//   ci_12xlarge: 1781.25,
+//   ci_16xlarge: 2375,
+// }
+
+export const COMPUTE_BASELINE_IOPS = {
+  ci_nano: 250,
+  ci_micro: 500,
+  ci_small: 1000,
+  ci_medium: 2000,
+  ci_large: 3600,
+  ci_xlarge: 6000,
+  ci_2xlarge: 12000,
+  ci_4xlarge: 20000,
+  ci_8xlarge: 40000,
+  ci_12xlarge: 50000,
+  ci_16xlarge: 80000,
+}
+export const COMPUTE_MAX_IOPS = {
+  ci_nano: 11800,
+  ci_micro: 11800,
+  ci_small: 11800,
+  ci_medium: 11800,
+  ci_large: 20000,
+  ci_xlarge: 20000,
+  ci_2xlarge: 20000,
+  ci_4xlarge: 20000,
+  ci_8xlarge: 40000,
+  ci_12xlarge: 50000,
+  ci_16xlarge: 80000,
+}
+export const COMPUTE_BASELINE_THROUGHPUT = {
+  ci_nano: 43,
+  ci_micro: 87,
+  ci_small: 174,
+  ci_medium: 347,
+  ci_large: 630,
+  ci_xlarge: 1188,
+  ci_2xlarge: 2375,
+  ci_4xlarge: 4750,
+  ci_8xlarge: 9500,
+  ci_12xlarge: 14250,
+  ci_16xlarge: 19000,
+}
+export const COMPUTE_MAX_THROUGHPUT = {
+  ci_nano: 2085,
+  ci_micro: 2085,
+  ci_small: 2085,
+  ci_medium: 2085,
+  ci_large: 4750,
+  ci_xlarge: 4750,
+  ci_2xlarge: 4750,
+  ci_4xlarge: 4750,
+  ci_8xlarge: 9500,
+  ci_12xlarge: 14250,
+  ci_16xlarge: 19000,
+}
+
+export const RESTRICTED_COMPUTE_FOR_IOPS_ON_GP3 = ['ci_nano', 'ci_micro', 'ci_small', 'ci_medium']
+
+export const RESTRICTED_COMPUTE_FOR_THROUGHPUT_ON_GP3 = [
+  'ci_nano',
+  'ci_micro',
+  'ci_small',
+  'ci_medium',
+]
+
+export const DISK_TYPE_OPTIONS = [
```

Comment:
product feedback, not ui feedback: 

This doesn't really tell me what I should do here. Is High Performance generally just for massive customers, etc...

![screenshot-2024-10-30-at-15 04 56](https://github.com/user-attachments/assets/be39b152-b3fb-45e6-aa9c-0837e0aecd81)

Reply:
@ greg (whats his username..) - we probably should link to docs here for each of these ?

Reply:
@gregnr

Reply:
Makes sense 👍 @charislam any thoughts?